### PR TITLE
Update doc to clarify the behavior when removing key/value pairs during iteration

### PR DESCRIPTION
**Pull request by ahrtr**
Refer to https://github.com/etcd-io/bbolt/pull/611

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -445,6 +445,11 @@ When you have iterated to the end of the cursor then `Next()` will return a
 before calling `Next()` or `Prev()`. If you do not seek to a position then
 these functions will return a `nil` key.
 
+If you remove key/value pairs during iteration, the cursor may automatically
```
Comment (fuweid) [inline README.md:448]: What about adding it as known issue ？

Comment (ahrtr) [reply README.md:448]: added.

**Comment by ahrtr** [comment]
@fuweid @tjungblu PTAL

**Review by fuweid — APPROVED** [review]
LGTM

**Pull request merged**